README.md

```diff
@@ -1,78 +1,59 @@
 # System Requirements
 
-- Install the latest Xcode (11.4 or higher) in `/Applications`, open it and make sure to install additional components if it asks you.
+- Install Xcode 11.5 in `/Applications`, open it and make sure to install additional components if it asks you.
 - Install [Carthage](https://github.com/Carthage/Carthage) if you haven't already. Using `brew` is recommended.
 
-# Building With Adobe DRM
-
-## Building the Application
+# Building without Adobe DRM nor Private Repos
 
-01. Contact project lead and ensure you have repo access to all required submodules, including private ones. Also request a copy of the Adobe RMSDK archive, which is currently not on Github, unzip it and place it in a place of your choice.
-02. Then run:
 ```bash
 git clone git@github.com:NYPL-Simplified/Simplified-iOS.git
-git clone git@github.com:NYPL-Simplified/Certificates.git
 cd Simplified-iOS
-ln -s <rmsdk_path>/DRM_Connector_Prerelease adobe-rmsdk
 git checkout develop
-git submodule update --init --recursive
+
+# one-time set-up
+./scripts/setup-repo-nodrm.sh
+
+# idempotent script to rebuild all dependencies
+./scripts/build-3rd-parties-dependencies.sh --no-private
```

Comment:
This should be `./scripts/build-3rd-party-dependencies.sh` I believe

Reply:
good catch, fixed